node/src/chain/acceptor.rs

```diff
@@ -86,6 +88,7 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
             vm: vm.clone(),
             network: network.clone(),
             task: RwLock::new(Task::new_with_keys(keys_path.to_owned())),
+            last_finalized: RwLock::new(last_finalized.clone()),
```

Comment:
Not in the scope of this review, since this mechanism is widely used all over the code base. But if the method need `last_finalized` just for cloning, then it would be better to take the ownership instead of reference

Reply:
Indeed.